client.go

```diff
@@ -6,37 +6,47 @@ import (
 	"errors"
 	"io/ioutil"
 	"net/http"
+	"net/url"
 	"strings"
 	"time"
 )
 
+// Client executes JSON RPC calls to remote servers.
 type Client struct {
-	Url     string
+	URL     string
 	Timeout int
+
+	proxyAddress string
```

Comment:
Questa variabile chiamiamola proxyURL. Poi con un refactor successivo renderei privata la variabile URL attuale e la rinominerei in serverURL per avere una certa uniformità.